WORKSPACE

```diff
@@ -1,5 +1,16 @@
 workspace(name = "io_bazel_rules_scala")
 
+
+####### previous rules scala for building of scala compiler plugin
+rules_scala_version="3070353d06bb726141e6cea241e5d6e355a4d14f" # update this as needed
```

Comment:
I don't like this approach here. It makes it more difficult to have your own branch of rules.

Could we instead have a `bootstrap` compiler mode that is very simple, but possibly slow that does not depend on fancy features? Or maybe just an option we can pass into the skylark which can use the direct/indirect approach at all?